### ISSUE TITLE
detach onDestroy

### ISSUE THREAD

**Issue by c00** [ladderio/ngx-tribute#9]
Currently the tribute object isn't properly disposed of when the directive is destroyed. 

This causes the DOM to get polluted with multiple `div.tribute-container` elements that are no longer used after the directive is destroyed.

The fix would be to implement the following in the directive:
```
ngOnDestroy(){
  this.tribute.detach(this.element.nativeElement);
}
```

**Comment by agarbund** [comment]
Thanks for reporting. Will try to tackle it this week. PR welcomed if you'd have time ;)

**Comment by c00** [comment]
PR added!

**Comment by agarbund** [comment]
Merged and published as 1.2.0
